src/Chapter3.hs

```diff
@@ -1006,6 +1132,31 @@ Implement data types and typeclasses, describing such a battle between two
 contestants, and write a function that decides the outcome of a fight!
 -}
 
+newtype Health2 = Health2 Int
+newtype Attack2 = Attack2 Int
+newtype Defence2 = Defence2 Int
```

Comment:
If you want, you can reuse data types from the previous task with newtypes :slightly_smiling_face: